index.html

```diff
@@ -7,7 +7,20 @@
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <div class="container"></div>
+    <div class="container">
+      
+      <h2>대기중 (<span id="count-waiting"></span>)</h2>
+      <ul class="list waiting"></ul>
+
+      <h2>완료됨 (<span id="count-complete"></span>)</h2>
+      <ul class="list complete"></ul>
+
+      <form class="input">
+        <input id="todo-new" type="text" placeholder="새로운 할일 추가">
+        <input id="button-submit" type="submit" value="+">
```

Comment:
조금더 명확한 네이밍을 사용하시면 좋을 것 같아요. 버튼인것은 알 수 있으니, 어떤 "기능"인지가 돋보이도록 `todo-submit`은 어떨까요?
```suggestion
        <input id="todo-submit" type="submit" value="+">
```

Reply:
<input id="button-submit" type="submit" value="+">
이 줄을 보고 버튼인 것을 알 수 있나요 ?
저도 submit-button 이런식으로 했을 것 같아요..

Reply:
네 ! 그게 더 좋을 것 같아요 :)